src/main/steps/applicant/support-you-need-during-case/SupportYouNeedDuringCaseController.ts

```diff
@@ -85,4 +50,46 @@ export class SupportYouNeedDuringYourCaseController extends PostController<AnyOb
       throw new Error('SafetyConcernsPostController - Case could not be updated.');
     }
   }
+
+  private getSupportYouNeedForApplicant(req: AppRequest<Partial<Case>>) {
+    if ('C100' === req.session.userCase.caseTypeOfApplication) {
+      req.session.userCase?.applicants?.forEach((applicant: Applicant) => {
```

Comment:
Why we need a forloop if we know only max of one idam user can be present in the list. Reduce the run time complexity .

Reply:
As for that c100 case there will be multiple applicants and and which is an array type. So in order to fetch the correct applicant we can use either forEach or for-of loops for that js array.